### PR TITLE
Minor coverage maintenance

### PR DESCRIPTION
**Pull request by cgewecke**
PR:
+ Updates coverage badge in README to track the master branch.
+ Restores mistakenly omitted testrpc server shutdown logic to the coverage shell script
+ .gitignores file debris solidity-coverage@0.1.7 generates when it crashes locally.

### CODE REVIEW

**Comment by cgewecke** [comment]
Please let me know if I should modify `coverage.sh` to reflect change proposed in #321.

**Comment by maraoz** [comment]
LGTM!

File: scripts/coverage.sh [hunk 1]
```diff
@@ -22,4 +20,7 @@ if [ ! $trpc_running ]; then
   trpc_pid=$!
 fi
 SOLIDITY_COVERAGE=true && ./node_modules/.bin/solidity-coverage
```
Comment (frangio) [inline scripts/coverage.sh]: Not introduced in this PR, but this line defines a variable that is not exported. Was the intention to define an env var for the `solidity-coverage` process? If so, `&&` should be removed.

**Comment by frangio** [comment]
Please wait until #325 is merged and update coverage.sh according to that. 😄

File: scripts/coverage.sh (hunk 1 above)
Comment (cgewecke) [inline scripts/coverage.sh]: Yes that's correct - it prevents HDWallet from being instantiated in `truffle.js` when coverage runs. Should just read as below?
```
SOLIDITY_COVERAGE=true  ./node_modules/.bin/solidity-coverage
```

Comment (frangio) [inline scripts/coverage.sh]: Yes, exactly.

**Comment by frangio** [comment]
#325 merged: [scripts/test.sh](https://github.com/OpenZeppelin/zeppelin-solidity/blob/86a69fd6d63279d2043ed1ff6f2d12598c0a9dbe/scripts/test.sh)

**Comment by cgewecke** [comment]
@frangio Updated, thanks for reviewing / improving!

**Comment by frangio** [comment]
Thanks to you for contributing!

**Pull request merged**